content/guides/building/get-source-git.md

```diff
@@ -48,64 +48,75 @@ to the last 10 commits, for example.
+</div>
+<p>This can happen if the username on your Gerrit account does not match the username that you use to log in to your computer. This would be the case in Haiku itself, as the default username is <code class="code">user</code>. You can make your life easier by <a href="https://linux.die.net/man/5/ssh_config">modifying your SSH configuration</a>.</p>
+<p><code class="varname">$USER</code> is an environment variable that is specific to Unix-like operating systems (e.g. Haiku, Linux, FreeBSD, etc.). If your system does not fall under that category, you will probably have to replace it.</p>
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
+<h4>Build Tools:</h4>
 
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/buildtools" && scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
+git clone "ssh://$USER@git.haiku-os.org/buildtools" && scp -p $USER@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
 ```
 
 <h4>Haiku:</h4>
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
-
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/haiku"
-scp -p <login>@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
+git clone "ssh://$USER@git.haiku-os.org/haiku" && scp -p $USER@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
 ```
 
-<h4>Switching from read-only to write access</h4>
+<h4>Sending in your first patch</h4>
+
+So, you want to contribute to Haiku. That's awesome, thank you for your interest!
 
-Want to submit your first patch? Thanks, that's great! You don't need to checkout the sources again. Instead you can update your existing copy of the source to use the commiter access. Just change the remote URL:
+<div class="alert alert-warning">
+Pull requests for the Haiku operating system made using GitHub cannot be accepted.
+</div>
+
+If you only have experience with GitHub and downloaded the source code of Haiku from it, don't worry, we've got you.
+
+First of all, you won't have to download everything all over again. You can just change the remote URL of your local copy with the following command:
 
 ```sh
-git remote set-url origin ssh://<login>@git.haiku-os.org/haiku
+git remote set-url origin ssh://$USER@git.haiku-os.org/haiku
 ```
 
-You also should install the <a href="https://review.haiku-os.org/Documentation/user-changeid.html">Gerrit hooks to generate Change-Ids</a>:
+You also should install a Git hook that will automatically <a href="https://review.haiku-os.org/Documentation/user-changeid.html">generate Change-Id's</a> for you:
 
 ```sh
-scp -p <login>@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
+scp -p $USER@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
 ```
 
+You should definitely do that, by the way. If a maintainer asks you to correct something later on, Gerrit will use that Change-Id to keep track of your newest changes.
```

Comment:
here as well I think the tone is a bit too familiar. Also, we should really just make this mandatory in Gerrit config, or update our Gerrit to a version that's not broken where the Change-Id is visible on the server side when Gerrit generates it.

Reply:
> Also, we should really just make this mandatory in Gerrit config.

My changes here were really based after real mistakes I made myself, this is off-topic but you should absolutely go for that.

Reply:
> You should definitely do that, by the way.

is indeed way too direct and friendly, but should the "you" pronoun/addressing the reader be done, in your opinion?